package.json

```diff
@@ -6,17 +6,17 @@
   "module": "lib/esm/insights-client.js",
```

Comment:
Bump the version to 0.3.0 in package.json.

Reply:
bumped